miraheze/puppet/upgrade-cp-service.py

```
@@ -0,0 +1,141 @@
+        else:
+            print(f"Health check failed for {server}")
+            input('Press enter to continue')
+    else:
+        print(f"Skipping health check as {server} did not respond to ping.")
+        input('Press enter to continue')
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('--servers', dest='servers', action=ServersAction, required=True, help='server(s) to deploy to')
+    args = parser.parse_args()
+    for server in servers:
```

Ruff (0.8.2)

Loop control variable `server` not used within loop body

Rename unused `server` to `_server`

(B007)